src/net/data_stream.rs

```diff
@@ -46,7 +46,8 @@ const HEADER_SIZE: usize = 8;
 const FRAME_COUNT: usize = 4;
 
 // The size of each livestream frame in bytes.
-const FRAME_SIZE: usize = 1024 + HEADER_SIZE;
+// Ensure this is below the network MTU (e.g. 1500 - ip and udp header)
+const FRAME_SIZE: usize = 4 * 8 * 2 * 22 + HEADER_SIZE;
```

Comment:
I'm not following what these magic numbers (4, 8, 2, 22) are supposed to be. Can we either document and/or break them out into their own configurations?

Also, there's a downside of using multiplication here in that it's hard to obviously see if the MTU size is violated. Perhaps we should add a static, compile-time assert on the frame size.

Reply:
Changed to count down from MTU size.